src/scripts/temporaryBattle/TemporaryBattleRunner.ts

```diff
@@ -19,6 +19,9 @@ class TemporaryBattleRunner {
         this.timeLeft(GameConstants.TEMP_BATTLE_TIME * this.timeBonus());
         this.timeLeftPercentage(100);
 
+        player.route(0);
+        Battle.route = 0;
+        Battle.catching(!(battle.optionalArgs.isTrainerBattle ?? true));
```

Comment:
I'm not quite sure what this part is supposed to be doing?
Could we add a comment explaining please